src/masonite/drivers/mail/BaseMailDriver.py

```diff
@@ -27,10 +29,11 @@ def __init__(self, app: App):
         self.to_addresses = []
         self.message_subject = 'Subject'
         self.message_reply_to = None
-        self.message_body = None
         self.from_name = self.config.FROM['name']
         self.from_address = self.config.FROM['address']
         self._queue = False
+        self.html_content = None
+        self.text_content = None
```

Comment:
We can do this and maintain the `message_body` attribute like we had. Simply manage a new `_mimetype` attribute instead